src/PhpSpec/CodeGenerator/Generator/Argument/StringBuilder.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PhpSpec\CodeGenerator\Generator\Argument;
+
+class StringBuilder
```

Comment:
I'd rather not use the `Builder` suffix when not implementing the builder pattern. The responsibility of this class seems to be something around converting reflection parameters into an arguments string, right?

(this will potentially be useful if we re-introduce argument type hints in future)